config/speechConfig.js.example

```diff
@@ -22,4 +22,14 @@ module.exports = {
   files: {
     uploadDirectory: null,  // enter file path to where uploads/publishes should be stored
   },
+  site: {
+    name       : 'Spee.ch',
+    host       : 'https://spee.ch',
+    description: 'Decentralized video and content hosting.',
+  },
+  publishing: {
+    defaultTitle      : 'Spee.h',
+    defaultThumbnail  : 'https://spee.ch/assets/img/video_thumb_default.png',
+    defaultDescription: 'Open-source, decentralized image and video sharing.',
```

Comment:
Where are both of these descriptions used?

Reply:
publishing.defaultDescription is used for open graph description on claims that do not have a description field.  stite.description was intended to be used wherever a description of the site itself was needed, but it turns out it wasn't needed so I removed it for simplicity (latest commit)